src/content/advanced/high-availability/control-plane/index.md

```diff
@@ -1,75 +1,78 @@
 ---
-linkTitle: High-availability masters
-title: High-availability Kubernetes masters
-description: A general description of high availability of masters as a concept, it's benefits, and some details you should be aware of.
+linkTitle: High-availability control plane
+title: High-availability Kubernetes control plane
+description: For production clusters with high availability requirements, Giant Swarm on AWS enables control planes with three control plane nodes and three etcd replicas spread over multiple availability zones. Here we explain most details.
```

Comment:
I thought about alternative wordings but think it just sounds cleanest without. If it's important to have a quasi-CTA there, I'd suggest "Details here"/"Details inside" or moving it to the beginning of the sentence: "For production clusters with high availability requirements, this article covers configuring clusters with three control plane nodes and `etcd` replicas spread over multiple availability zones".
```suggestion
description: For production clusters with high availability requirements, Giant Swarm on AWS enables control planes with three control plane nodes and three etcd replicas spread over multiple availability zones.
```